tools/experiments/experiments.html

```diff
@@ -212,6 +212,11 @@ <h2>Release channels</h2>
   </section>
 
   <section class="content">
+      <p>
+        <em>Note:</em> these controls set a cookie on the domain name <code>cdn.ampproject.org</code>. If your browser is
+        set to block all third-party cookies, they will not function. Make sure you disable third-party cookie blocking,
+        or whitelist the websites that you are testing to allow for them.
+      </p>
```

Comment:
This `<p>` is right now a part of the `table` and won't pick up the correct formatting. Move it to after the previous `<p>` block (lines 208-211), inside the `<section class="desc">` (so that formatting is picked up).

Reply:
Done